_posts/2020-08-26-#19339.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: pr
+date: 2020-08-26
+title: "Redelgate AbsurdFee from Mempool to Clients"
+pr: 19339
+authors: [gzhao408]
+components: ["validation", "mempool", "rpc"]
```

Comment:
The labels are also different from those of the PR (and `rpc` isn't a valid label, currently). If these were changed because the PR is labeled incorrectly, the labels should still be Bitcoin Core ones; see https://github.com/bitcoin/bitcoin/labels and https://github.com/bitcoin-core-review-club/website/blob/master/Rakefile#L25. From looking at the PR, however, I'd just use the actual labels (except `Review club`).

Here is what I get when running the script...
```
$ rake posts:new -- -p 19339 -h gzhao408 -d 2020-08-26
GitHub PR title:  "validation: re-delegate absurd fee checking from mempool to clients"
Parsed PR title:  "Re-delegate absurd fee checking from mempool to clients"
GitHub PR labels: "rpc/rest/zmq, review club, tests, validation, wallet"
Parsed PR labels: "rpc/rest/zmq, tests, validation, wallet"
New file _posts/2020-08-26-#19339.md created successfully.
```
...which generates this post metadata...
```
layout: pr
date: 2020-08-26
title: "Re-delegate absurd fee checking from mempool to clients"
pr: 19339
authors: [gzhao408]
components: ["rpc/rest/zmq", "tests", "validation", "wallet"]
host: gzhao408
.../...
```

Reply:
ohhh, this is helpful! I had written it manually 😅

Reply:
great! can just paste the title and components from this, then :)